Сhapter4/Chapter4.Task9/Program.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Drawing;
+//9. Створити структуру Point яка складається з координат X i Y(два числові поля). Створити структуру Rectangle, яка складається з двох точок(ліва нижня і права верхня). Написати методи для знаходження площі і периметру прямокутника. Формула для знаходження відстані між точкою А і В. AB = √(xb - xa)2 + (yb - ya)2
+namespace Chapter4.Task9
+{
+    class Program
+    {
+        struct Point
+        {
+            public int X;
+            public int Y;
+
+            public Point(int XPos, int YPos)
+            {
+                X = XPos;
+                Y = YPos;
+            }
+
+            public void Display()
+            {
+                Console.WriteLine($"x = {X}, y = {Y}");
+            }
+        }
+        struct Rectangle
+        {
+            public Point topRight;
+            public Point bottomLeft;
+            public Point topLeft;
+            public Point bottomRight;
+            public int P;
+            public int S;
+            public void DisplayRect()
+            {
+                Console.WriteLine($"Perimetr = {P},Square = { S}");
+            }
+        }
+        static void Main(string[] args)
+        {
+            Point A = new Point(2, 4);
+            A.Display();
+            Point C = new Point(21, 17);
+            C.Display();
+
+            Rectangle rect = new Rectangle();
+
```

Comment:
Make parametrized constructor for Rectangle. Pass there bottom left and top right points(only 2 points!!!) other 2 you should calculate as you are doing on 46-47 lines.